app/controllers/redirect_controller.rb

```diff
@@ -91,7 +91,7 @@ def render_410
   end
 
   def find_id(uuid)
-    solr_rsp =  ActiveFedora::SolrService.instance.conn.get 'select', :params => {:q => Solrizer.solr_name('fedora3uuid')+':'+uuid}
+    solr_rsp =  ActiveFedora::SolrService.instance.conn.get 'select', :params => {:q => 'fedora3uuid_ssim:'+uuid}
```

Comment:
I'm going to change ```fedora3uuid``` and ```ark_id``` to ```Solrizer.solr_name('fedora3uuid', :symbol)``` so that we keep Solrizer responsible for managing [suffixes](http://www.rubydoc.info/gems/solrizer/3.3.0/Solrizer/DefaultDescriptors).